tests/integration/test_passwords.py

```diff
@@ -216,6 +216,22 @@ def test_reset_password_post_success(self, mock_request):
         self.assertEqual(response.status_code, 200)
         self.assertTrue("Your password has been changed".encode() in response.data)
 
+    @requests_mock.mock()
+    def test_reset_password_respondent_not_found(self, mock_request):
+        mock_request.get(url_banner_api, status_code=404)
+        mock_request.put(url_password_change, status_code=200)
```

Comment:
Do we need to mock url_password_change? Might also look a bit neater if the mocks were all together

Reply:
Removed and moved order of statements